resources/css/functions.less

```diff
@@ -118,7 +118,8 @@
 }
 
 .inner-shadow-top() {
-  box-shadow+: inset 0 (@box-shadow-radius - 1px) @box-shadow-radius -1px @box-shadow-color;
+  box-shadow: inset 0 (@box-shadow-radius - 1px) @box-shadow-radius -1px
```

Comment:
that's not right...

Reply:
whoops deleted it while testing something else 👀